File: src/main/java/com/kkllffaa/meteor_litematica_printer/Modules/CRUD/AtomicSettings/ContainerSettings.java
package com.kkllffaa.meteor_litematica_printer.Modules.CRUD.AtomicSettings;

import com.kkllffaa.meteor_litematica_printer.Addon;

import meteordevelopment.meteorclient.settings.SettingGroup;
import meteordevelopment.meteorclient.systems.modules.Module;

public class ContainerSettings extends Module {
    public static ContainerSettings Instance = new ContainerSettings();
    public ContainerSettings() {
		super(Addon.SettingsForCRUD, "Container", "Module to configure AtomicSettings.");
		this.toggle();
	}

	@Override
	public void toggle() {
        if (isActive()) {
            return;
        }
        super.toggle();
    }

	
	private final SettingGroup sgGeneral = settings.getDefaultGroup();
}

File: src/main/kotlin/com/kkllffaa/meteor_litematica_printer/Modules/CRUD/AtomicSettings/ContainerSettings.kt
package com.kkllffaa.meteor_litematica_printer.Modules.CRUD.AtomicSettings

import com.kkllffaa.meteor_litematica_printer.Addon
import meteordevelopment.meteorclient.settings.SettingGroup
import meteordevelopment.meteorclient.systems.modules.Module

class ContainerSettings : Module(Addon.SettingsForCRUD, "Container", "Module to configure AtomicSettings.") {
    override fun toggle() {
        if (isActive()) {
            return
        }
        super.toggle()
    }


    private val sgGeneral: SettingGroup? = settings.getDefaultGroup()

    init {
        this.toggle()
    }

    companion object {
        var Instance: ContainerSettings = ContainerSettings()
    }
}
